okio/src/jvmMain/kotlin/okio/Pipe.kt

```diff
@@ -15,8 +15,6 @@
  */
 package okio
 
-import java.io.IOException
```

Comment:
Why?

Reply:
We’ve got our own `okio.IOException` that’s aliased to `java.io.IOException`. No behavior change here.